extensions/amp-social-share/1.0/amp-social-share.js

```diff
@@ -238,8 +238,8 @@ AmpSocialShare['delegatesFocus'] = true;
 
 /** @override */
 AmpSocialShare['props'] = {
-  'tabIndex': {attr: 'tabindex'},
-  'type': {attr: 'type'},
+  'tabIndex': {attr: 'tabindex', media: true},
```

Comment:
Ditto. We shouldn't have media on `tabIndex` to keep it more compatible with Web concepts.

Reply:
`tabindex` is all lowercase for HTML: https://html.spec.whatwg.org/multipage/interaction.html#attr-tabindex

Reply:
React mentions this here as well: https://reactjs.org/docs/dom-elements.html